docs/en/02_Developer_Guides/06_Testing/index.md

```diff
@@ -19,22 +19,16 @@ If you are familiar with PHP coding but new to unit testing then check out Mark'
 You should also read over [the PHPUnit manual](http://www.phpunit.de/manual/current/en/). It provides a lot of
 fundamental concepts that we build on in this documentation.
 
-Unit tests are not included in the normal SilverStripe downloads so you need to install them through git repositories
-([installation instructions](/getting_started/composer)).
+Unit tests are not included in the zip/tar.gz SilverStripe [downloads](http://www.silverstripe.org/software/download/) so to get them, install SilverStripe [with composer](/getting_started/composer).
 
-## Install with Composer
+## Invoking phpunit
 
-Once you've got your project up and running, open a terminal and cd to your project root.
+Once you have used composer to create your project, `cd` to your project root. Composer will have installed PHPUnit alongside the required PHP classes into the `vendor/bin/` directory.
 
-	composer require --dev "phpunit/phpunit:*@stable"
+If you don't want to invoke PHPUnit through its full path (`vendor/bin/phpunit`), add `./vendor/bin` to your $PATH, or symlink phpunit into the root directory of your website:
```

Comment:
What about the case where some non-SilverStripe project needs a different version of phpunit? Doesn't it make more sense to tie each project to its preferred phpunit? I can see your way making sense in a homogeneous environment. What do you think about presenting both alternatives?